spring-kafka/src/test/java/org/springframework/kafka/core/KafkaTemplateTests.java

```diff
@@ -265,6 +267,29 @@ public boolean isInterestedInSuccess() {
 	}
 
 	@Test
+	public void withCallback() throws Exception {
```

Comment:
We might need to have similar test in the `KafkaTemplateTransactionTests` to be sure that TX behavior remains